### ISSUE TITLE
Show version of the app.

### ISSUE THREAD

**Issue by diogodias86** [cejug/hurraa#119]
show the version number of the app on the pages.

**Comment by diogodias86** [comment]
what version number is to show? the value of pom.xml?

**Comment by efraimgentil** [comment]
Yeah, the version on the pom would be nice, you can see this discussion in the group,
https://groups.google.com/forum/#!topic/cejug/1R_muYDrmQw
it may help you
